### PR TITLE
Added Bip deployment guide

### PR DESCRIPTION
**Pull request by joelkennedy**
Hello there!

I have updated the one-command deployment doc to include a guide on how to deploy to [Bip](https://bip.sh).

Unfortunately I was unable to translate the guide into the other supported languages. Looking at a similar documentation PR raised yesterday, having followed their lead, I have copied the text to the other languages. I hope that's OK.

## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)
    - Languages:
    - [x] `en` English
    - [x] `ko` Korean
    - [x] `pt-br` Brazilian Portuguese
    - [x] `ru` Russian
    - [x] `th` Thai
    - [x] `zh-cn` simplified Chinese
    - [x] `zh-tw` traditional Chinese

### CODE REVIEW

**Comment by SukkaW** [comment]
> Unfortunately I was unable to translate the guide into the other supported languages. Looking at a similar documentation PR raised yesterday, having followed their lead, I have copied the text to the other languages. I hope that's OK.

That will be ok then. Translation will be brought up by translators later.

File: source/docs/one-command-deployment.md [hunk 1]
```diff
@@ -254,6 +254,28 @@ The sample app for `hexo` is deployed at [https://hexo.21yunbox.com/](https://he
 
 For more detail, follow this guide at [https://www.21yunbox.com/docs/#/deploy-hexo](https://www.21yunbox.com/docs/#/deploy-hexo).
 
+## Bip
+
+[Bip](https://bip.sh) provides zero downtime deployment, a global CDN, SSL, unlimited bandwidth and more for static websites.
```
Comment (SukkaW) [inline source/docs/one-command-deployment.md:259]: It appears that `Bip` is not a free service. IMHO it should be mentioned in the documents before `bip init`.

Comment (joelkennedy) [reply source/docs/one-command-deployment.md]: @SukkaW sure. I've updated the docs to note right upfront that it's a commercial service, and have included the pricing structure (pay as you go, per domain). Let me know if that's OK.

**Review by SukkaW — APPROVED** [review]

File: source/docs/one-command-deployment.md (hunk 1 above)
Comment (SukkaW) [reply source/docs/one-command-deployment.md]: LGTM!

**Comment by stevenjoezhang** [comment]
@joelkennedy bip.sh is no longer accessible. Can you help confirm if it's still working?

**Comment by joelkennedy** [comment]
@stevenjoezhang we had a slight issue affecting our main website but it's now been resolved. Sorry about that - it's now working.

**Pull request merged**